perf/benchmarkapps/GrpcAspNetCoreServer/Program.cs

```diff
@@ -93,6 +84,24 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                             ConfigureListenOptions(listenOptions, config, endPoint);
                         });
                     }
+                    else if (!string.IsNullOrEmpty(namedPipeName))
+                    {
+                        Console.WriteLine($"Named pipe name: {namedPipeName}");
+                        options.ListenNamedPipe(namedPipeName, listenOptions =>
+                        {
+                            ConfigureListenOptions(listenOptions, config, endPoint);
+                        });
+                    }
+                    else
+                    {
+                        // ListenAnyIP will work with IPv4 and IPv6.
+                        // Chosen over Listen+IPAddress.Loopback, which would have a 2 second delay when
```

Comment:
Is this in the aspnetcore docs somewhere?

Reply:
No. This is just what I observed on my computer a few years ago, and this change fixed it.